include/tss2/tss2_esys.h

```diff
@@ -309,8 +309,7 @@ Esys_StartAuthSession(
     TPM2_SE sessionType,
```

Comment:
I'm not sure if I'm looking at the right spec, but according to the latest I could find Esys_StartAuthSession() function also takes TPM2B_ENCRYPTED_SECRET const *encryptedSalt,
See https://trustedcomputinggroup.org/wp-content/uploads/TSS_TSS-2.0-Enhanced-System-API_V0.9_R03_Public-Review-1.pdf
section 11.3.6 Esys_StartAuthSession.

Reply:
latest is https://trustedcomputinggroup.org/wp-content/uploads/TSS_ESAPI_Version-0.9_Revision-04_reviewEND030918.pdf